Sources/TuistKit/ProjectEditor/ProjectEditor.swift

```diff
@@ -93,6 +93,7 @@ final class ProjectEditor: ProjectEditing {
         let projectDesciptionPath = try resourceLocator.projectDescription()
         let manifests = manifestFilesLocator.locateAllProjectManifests(at: at)
         let configPath = manifestFilesLocator.locateConfig(at: at)
+        let dependenciesPath = manifestFilesLocator.locateDependencies(at: at)
```

Comment:
Not related ot this PR, but maybe we can add a parameter name and an argument label so this reads like:
```
func edit(at editingPath: AbsolutePath, in dstDirectory: AbsolutePath) throws -> AbsolutePath {
    ...
    let dependenciesPath = manifestFilesLocator.locateDependencies(at: edititingPath)

}
```